### ISSUE TITLE
Syntax warning

### ISSUE THREAD

**Issue by fatuuse** [hasl-sensor/integration#7]
Not important, typo?

```
2021-06-12 21:01:44 WARNING (MainThread) [homeassistant.helpers.entity] Update of sensor.signal_messages is taking over 10 seconds
/config/custom_components/hasl3/sensor.py:370: SyntaxWarning: "is" with a literal. Did you mean "=="?
  if sensorproperty is 'min':
/config/custom_components/hasl3/sensor.py:380: SyntaxWarning: "is" with a literal. Did you mean "=="?
  if sensorproperty is 'time':
/config/custom_components/hasl3/sensor.py:389: SyntaxWarning: "is" with a literal. Did you mean "=="?
  if sensorproperty is 'deviations':
/config/custom_components/hasl3/sensor.py:392: SyntaxWarning: "is" with a literal. Did you mean "=="?
  if sensorproperty is 'updated':
/config/custom_components/hasl3/sensor.py:447: SyntaxWarning: "is not" with a literal. Did you mean "!="?
  if self._unit_of_measure is not '':
```

**Comment by DSorlov** [comment]
Perfect find! It is amazing how much I can miss sometimes. Will fix to next version which should be out in a few days.

**Comment by DSorlov** [comment]
Resolved in [3.0.0-beta.2](https://github.com/hasl-sensor/integration/releases/tag/3.0.0-beta.2)
